### PR TITLE
changes to ui panel add and remove links

### LINKED ISSUE #1457: Change add field link
As shown in https://github.com/LD4P/sinopia/issues/261#issuecomment-531882451.

Note that this is for "add", not "add another".

### PR DESCRIPTION
**Pull request by peetucket**
fixes #1457 and #1718 and #1455

### CODE REVIEW

**Comment by justinlittman** [comment]
@peetucket Can you attach a screen shot or two that @astridu can review?

**Comment by peetucket** [comment]
> @peetucket Can you attach a screen shot or two that @astridu can review?

Yeah, I did and attached to ticket #1457 -- and she reviewed there.

File: src/components/editor/property/OutlineHeader.jsx [hunk 1]
```diff
@@ -43,12 +44,12 @@ const OutlineHeader = (props) => {
...
       <button type="button" className="btn btn-sm btn-toggle" onClick={props.handleToggle} data-id={props.id} disabled={isAdd}>
-        <FontAwesomeIcon icon={icon} />
+        <FontAwesomeIcon id="toggle-icon" icon={toggleIcon} />
       </button>
       <strong><PropertyLabel propertyTemplate={props.property} /></strong>
       <PropertyLabelInfo propertyTemplate={ props.property } />
-      <button type="button" className="btn btn-sm btn-outline-primary btn-remove" onClick={props.handleRemoveButton} data-id={props.id}>
-        Remove
+      <button type="button" className="btn btn-sm btn-remove" onClick={props.handleRemoveButton} data-id={props.id}>
+        <FontAwesomeIcon id="trash-icon" icon={trashIcon} />
```
Comment (justinlittman) [inline src/components/editor/property/OutlineHeader.jsx]: id should be unique.

**Comment by justinlittman** [comment]
This doesn't look quite right:

image

**Comment by astridu** [comment]
I think Peter was going to leave the field name in the "+Add" links within the property so it should say "+Add {field.name}"

**Comment by peetucket** [comment]
> I think Peter was going to leave the field name in the "+Add" links within the property so it should say "+Add {field.name}"

Ahh, I didn't do that, but can definitely put it back.

File: src/components/editor/property/OutlineHeader.jsx (hunk 1 above)
Comment (peetucket) [reply src/components/editor/property/OutlineHeader.jsx]: HMM...the reason I did this was to get a hook on that icon for use in testing.  Maybe I can use a class instead.

Comment (peetucket) [reply src/components/editor/property/OutlineHeader.jsx]: removed the need for using an ID here

**Comment by peetucket** [comment]
Added back in the name of the field being added to the "+ Add" link

**Comment by astridu** [comment]
Sorry for the confusion!

**Comment by peetucket** [comment]
> Sorry for the confusion!

No problems!

**Comment by justinlittman** [comment]
I'd like to get it a bit more consistent:
image

@peetucket maybe try pairing with another dev? Layout in BS is a pain, so maybe another set of eyes will help.

**Comment by peetucket** [comment]
sure i am free today to pair on this

**Comment by peetucket** [comment]
Trash can icons right-aligned.  Switch Add and Remove buttons at field level to text link/icon respectively.  See screen shots below:

Screen Shot 2019-11-18 at 2 50 41 PM
Screen Shot 2019-11-18 at 2 50 36 PM

**Pull request merged**